windows/shared/logging/logging-plog-windows/include/logging/windows/PlogDebugWrapper.hxx

```diff
@@ -0,0 +1,41 @@
+
+#ifndef PLOG_DEBUG_WRAPPER_HXX
+#define PLOG_DEBUG_WRAPPER_HXX
+
+#include <windows.h>
+
+#include <plog/Appenders/DebugOutputAppender.h>
+#include <plog/Appenders/DynamicAppender.h>
+#include <plog/Log.h>
+
+#include <string>
+
+namespace logging
+{
+namespace plog
+{
+
+template <typename Formatter>
+class DebugWrapperAppender : public ::plog::DynamicAppender
```

Comment:
Why does this need to be a `DynamicAppender` instead of `IAppender`? My understanding is that `DynamicAppender` is needed for adding an appender safely at runtime after initialization, but I'm not sure we have any use case where that's needed. Do you have one in mind?